README.md

```diff
@@ -1,33 +1,82 @@
+ - Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+ - We respect the opinion of others and are gracious in our interactions, whether meeting physically or digitally, whether friend or stranger, whether employee, customer or commenter.
+ - Our experiences and backgrounds differ, and therefore so do our perspectives, areas of expertise, and levels of experience. All in all, we pull each other up, not bring each other down.
+ - We value and respect all differences in all people (seen and unseen). We aspire to inclusive working experiences and an environment that reflects the audience we serve, where our people take up career development opportunities equally, their voices are heard and contribute to our future. We aim to hire, retain and advance people from underrepresented groups.
 
 
-We believe that all of us want to - and can - make a difference to the Guardian.
+**Working as a team**
+
+We believe that the team is everything and our lowest common unit, set up with the necessary skills to succeed.
 
->We choose to work here because we believe in what the Guardian stands for. We're excited at contributing to the Guardian's future. 
+_Behaviours_:
+ - We fail and succeed together as a team.
+ - We are independent, accountable and share responsibility. We conduct ourselves with humility, kindness and a calm manner. We debate ideas, not people and we share constructive feedback openly. 
+ - We take the time needed to listen to and help our colleagues with any problem they might have from anywhere within the business.
+ - We value the experience of people in the team and the different constraints within their lives. 
+ - We celebrate success and positive behaviour, recognizing everyone’s contribution while treating every moment including tasks and failures as a learning opportunity.
 
-We believe in treating everyone we encounter with respect.
+**We are data informed** 
 
->Whether meeting physically or electronically, whether friend or stranger, whether employee, customer or commenter, we respect the opinion of others and are gracious in our interactions.
+We believe that to be responsible in our impact we must be informed by data.
 
-We are respectful of different skill sets and levels of ability.
->We come from different backgrounds, work in different roles, are different ages, and therefore have different levels of experience. All in all, we pull each other up, not bring each other down.
+_Behaviours_: 
+ - We base our decisions on analytics and experiments to reach consensus having heard everyone’s voice. 
+ - We measure our success through data analysis and not through the highest paid person's opinion or the loudest voice.
```

Comment:
This feels a bit disingenuous as it implies that data analysis is in some way objective. I suspect it also isn't true, the process of "data analysis" probably delivers something that allows a consensus to be generated about whether a change is successful.

Reply:
This is a fair point as objectivity is difficult(impossible) to achieve and one could always game a system to achieve what they would like. However I think this sentence is explicit enough on how we aim to measure success without defining a flawless protocol to do so.